autoinstrumentation/java/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM busybox
```

Comment:
We seem to be using this for the operator image:

https://github.com/open-telemetry/opentelemetry-operator/blob/c3f411cb6f839443354a2039a0d1190d269147bd/Dockerfile#L31